include/swupd-error.h

```diff
@@ -19,7 +19,7 @@
 #define EFULLDOWNLOAD 17	/* full_download problem */
 #define ENET404 404		/* download 404'd */
 #define EBUNDLE_INSTALL 18      /* Cannot install bundles */
-#define EREQUIRED_DIRS 19	/* Cannot create required dirs */
-#define ECURRENT_VERSION 20	/* Cannot determine current OS version */
+#define EREQUIRED_DIRS 19       /* Cannot create required dirs */
+#define ECURRENT_VERSION 20     /* Cannot determine current OS version */
```

Comment:
I am really confused by clang-format's decision here... Maybe it's only looking at EBUNDLE_INSTALL line? Regardless, I would prefer that these lines be tab-aligned.